buildconfig/pygame-stubs/mixer.pyi

```diff
@@ -34,17 +35,12 @@ def get_sdl_mixer_version(linked: bool) -> Tuple[int, int, int]: ...
 
 class Sound:
     @overload
-    def __init__(self, file: IO) -> None: ...
+    def __init__(self, file: IO[bytes]) -> None: ...
```

Comment:
Same question about `IO[bytes]` here as above.

Reply:
IO[AnyStr] can be str or bytes; since this is a sound file, I presuming it has to be bytes